### PR TITLE
doc: Remove latin from nrf connect sdk docs

### PR DESCRIPTION
**Pull request by divipillai**
NCSDK-12222
Remove Latin abbreviations from the
nrf connect sdk documents.

Signed-off-by: divya pillai <divya.pillai@nordicsemi.no>

### CODE REVIEW

File: applications/serial_lte_modem/doc/SOCKET_AT_commands.rst [hunk 1]
```diff
@@ -219,7 +219,7 @@ Syntax
 
 * The ``<sec_tag>`` parameter is an integer.
   It indicates to the modem the credential of the security tag to be used for establishing a secure connection.
-  It is associated with a credential, i.e. certificate or PSK. The credential should be stored on the modem side beforehand.
+  It is associated with a credential, that is, certificate or PSK. The credential should be stored on the modem side beforehand.
```
Comment (peknis) [inline applications/serial_lte_modem/doc/SOCKET_AT_commands.rst:222]:
```suggestion
  It is associated with a credential, that is, a certificate or PSK. The credential should be stored on the modem side beforehand.
```

File: doc/nrf/glossary.rst [hunk 2]
```diff
@@ -278,7 +278,7 @@ Glossary
       A machine-to-machine (M2M) connectivity protocol used by some IoT devices.
       It is designed as an extremely lightweight publish/subscribe messaging transport.
       It is useful for connections with remote locations where a small code footprint is required and/or network bandwidth is at a premium.
-      For example, it has been used in sensors communicating to a broker via satellite link, over occasional dial-up connections with healthcare providers, and in a range of home automation and small device scenarios.
+      For example, it has been used in sensors communicating to a broker through satellite link, over occasional dial-up connections with healthcare providers, and in a range of home automation and small device scenarios.
```
Comment (peknis) [inline doc/nrf/glossary.rst:281]:
```suggestion
      For example, it has been used in sensors communicating to a broker through a satellite link, over occasional dial-up connections with healthcare providers, and in a range of home automation and small device scenarios.
```

File: doc/nrf/libraries/caf/leds.rst [hunk 3]
```diff
@@ -13,7 +13,7 @@ The source of such events could be any other module in |NCS|.
 The module uses Zephyr's :ref:`zephyr:led_api` driver for setting the LED color, either RGB or monochromatic.
 Zephyr's LED driver can use the implementation based on either GPIO or PWM.
 Use the PWM-based implementation to achieve smooth changes of brightness.
-Use the GPIO-based implementation if your board does not support PWM or if you want to control LEDs connected via GPIO expander.
+Use the GPIO-based implementation if your board does not support PWM or if you want to control LEDs connected using GPIO expander.
```
Comment (peknis) [inline doc/nrf/libraries/caf/leds.rst:16]:
```suggestion
Use the GPIO-based implementation if your board does not support PWM or if you want to control LEDs connected using the GPIO expander.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Comment by pdunaj** [comment]
Why? These are standard abbreviations... beloved by everybody for millennia :(

**Comment by mbolivar-nordic** [comment]
> Why? These are standard abbreviations... beloved by everybody for millennia :(

"Via" is not even an abbreviation, acronym, or initialism. It's just a plain old word:

https://www.merriam-webster.com/dictionary/via

File: doc/nrf/ug_nrf5340.rst [hunk 4]
```diff
@@ -261,7 +261,7 @@ If you want to enable the network core anyway, set the :kconfig:`CONFIG_BOARD_EN
 
 .. _ug_nrf5340_multi_image:
 
-Single image vs. multi-image build
+Single image and multi-image build
```
Comment (mbolivar-nordic) [inline doc/nrf/ug_nrf5340.rst:264]: This is changing the meaning. The two types of build are mutually exclusive, hence "vs." (versus). Plain "and" does not convey this clearly.

**Comment by peknis** [comment]
> Why? These are standard abbreviations... beloved by everybody for millennia :(

Our "bible", Read this first, says: "Do not use Latin abbreviations..." Just for the consistency, we are cleaning up the docs, leaving all of them out. The same book also suggests avoiding any other words of foreign origin, such as "via".

Let's not go into deeper debate of the origin of words...

File: doc/nrf/ug_multiprotocol_support.rst [hunk 5]
```diff
@@ -15,7 +15,7 @@ For more information about how the SoftDevice Controller and MPSL work together,
 
 The multiprotocol solution available for Thread and Zigbee is dynamic.
 This means that it allows for running several radio protocols simultaneously without the time-expensive uninitialization and initialization in-between the switching.
-Switching between protocols requires only a reinitialization of the radio peripheral, since protocols may operate on different frequencies, modulations, etc.
+Switching between protocols requires only a reinitialization of the radio peripheral, since protocols may operate on different frequencies and modulations.
```
Comment (pdunaj) [inline doc/nrf/ug_multiprotocol_support.rst:18]: `requires only a reinitialization` seems strange to me. Is the `only` needed?

**Review by pdunaj — APPROVED** [review]

**Comment by pdunaj** [comment]
> Our "bible", Read this first, says: "Do not use Latin abbreviations..."

What communism has thought me is that you must use force to get rid of people natural behavior.
Since these Latin words are extremally popular in use, these will constantly pop up whatever "bible" states.

If this is really needed I suggest adding some word validation to checkpatch script (if it's not already there), and register all Latin words (as long as other unwanted words) so these will cause PR to be rejected by the CI.

**Review by MarekPorwisz — APPROVED** [review]

**Review by coderbyheart — APPROVED** [review]
**suggestion:** add a GitHub action that checks for forbidden latin words.

**Comment by b-gent** [comment]
> > Our "bible", Read this first, says: "Do not use Latin abbreviations..."
> 
> What communism has thought me is that you must use force to get rid of people natural behavior. Since these Latin words are extremally popular in use, these will constantly pop up whatever "bible" states.
> 
> If this is really needed I suggest adding some word validation to checkpatch script (if it's not already there), and register all Latin words (as long as other unwanted words) so these will cause PR to be rejected by the CI.

Hi Paweł, this is docs, tech writers are responsible for the styleguide and we will enforce it since there must be a tech writer on every 'doc-required' PR anyway.
Getting rid of Latin is nothing strange in technical communication, these expressions may seem natural to you and they have been 'beloved for millenia' from your cultural perspectieve but think, for example, about Asian customers for whom these might be cryptic.

File: doc/nrf/ug_nrf5340.rst (hunk 4 above)
Comment (divipillai) [reply doc/nrf/ug_nrf5340.rst:264]: Since the content mentions more about the Multi-image build, we could change the title to 'Multi-image build'.

Comment (peknis) [reply doc/nrf/ug_nrf5340.rst:264]: It mentions only the multi-image build. Not a word about single-image.

**Comment by mbolivar-nordic** [comment]
> The same book also suggests avoiding any other words of foreign origin, such as "via".

Perhaps the commit log message should reflect that it is abbreviations and loan words that are being removed, not just abbreviations, then? That would have resolved my confusion when first reading this.

File: doc/nrf/ug_nrf5340.rst (hunk 4 above)
Comment (mbolivar-nordic) [reply doc/nrf/ug_nrf5340.rst:264]: > we could change the title to 'Multi-image build'.

sounds reasonable!

**Pull request merged**